src/modules/goal/reducer.js

```diff
@@ -6,7 +6,7 @@ const initialState = {
   currentBalance: {
```

Comment:
I would use `null` instead of `undefined` when you want to declare a variable/property without a value. `undefined` is default, and means that a variable is declared but is missing the value. `null` means that the variable/property is set empty.